{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/context_processors.py

```diff
@@ -0,0 +1,8 @@
+from django.conf import settings
+
+
+def exposed_settings(request):
```

Comment:
```suggestion
def expose_settings(request):
```

Reply:
Sorry for the long lead time. This has now been addressed